packages/toaster/src/index.ts

```diff
@@ -1,3 +1,16 @@
 import Toaster from './toaster'
+import {
+  ToastsWrapper,
+  ToastWrapperAnimated,
+  ToastWrapper,
+  ToastAlert
+} from './styles'
 
+export {Toast} from './toast'
 export const toaster = new Toaster()
+export const ToastElements = {
+  ToastsWrapper,
```

Comment:
```suggestion
  Wrapper,
```
Remove prefixes `Toasts` and `Toast because they are in `ToastElements` which is kind of a prefix. Also you can try to give more meaningful names. Later this will be used in theme:

```tsx
export const customTheme= createTheme({
  customize: {
    toast: elements => css`
      ${elements.Wrapper} {
        padding: 16px;
      }
      ${elements.Alert} {
        margin-right: 12px;
      }
    `,
  },
})
```